search-service/src/main/kotlin/com/egm/stellio/search/service/EntityEventListenerService.kt

```diff
@@ -76,6 +69,59 @@ class EntityEventListenerService(
             logger.error(RECEIVED_NON_PARSEABLE_ENTITY, e)
         }
 
+    private fun handleEntityDeleteEvent(entityDeleteEvent: EntityDeleteEvent) =
+        temporalEntityService.deleteTemporalEntityReferences(
```

Comment:
it misses some consistency: you don't use the same service when creating the temporal entity references (`temporalEntityAttributeService`) and when deleting them (`temporalEntityService`)

Reply:
Updated [here](https://github.com/stellio-hub/stellio-context-broker/pull/398/commits/b73f9522e9199a2ed33196a54ce916a74bee112f).